### PR TITLE
Updated Patos bio

### PR DESCRIPTION
**Pull request by gecofernando**

### CODE REVIEW

File: source/team/lu.html.erb [hunk 1]
```diff
@@ -1,34 +0,0 @@
----
```
Comment (adamantivm) [inline source/team/lu.html.erb]: @gecofernando did you mean to remove Lucas?

Comment (gecofernando) [inline source/team/lu.html.erb]: Ups. Definitely not.

Comment (adamantivm) [inline source/team/lu.html.erb]: let me know when it's fixed.

Comment (gecofernando) [inline source/team/lu.html.erb]: Done!

**Comment by adamantivm** [comment]
deployed

**Pull request merged**